src/schemas/list.ts

```diff
@@ -0,0 +1,4 @@
+export interface ListOption {
+  pageSize: number,
+  currentPage: number,
```

Comment:
According to the Google Cloud API Design guides, where should we put the `next_page_token` in this payload?

Reply:
Don't agree, `next_page_token` is in response object. Use `pageToken` replace `currentPage`. 

And then I would add `startAfter`, it could make get list more flexibly.

```
interface ListOption {
  pageSize: number,
  pageToken?: number,
  startAfter?: string,
}
```

Reply:
`startAfter` is not a good name, because the list should be a [start, end) range. 

To include the start number, the name of `startFrom` would be better than `startAfter`.

Reply:
If we want to get the list which start from one specific `KEY`, how to do with `ListOption`?

For example, Moment timeline, when we want to get the next page, we should specific the last moment id as the `KEY`. 

Due to the moment timeline will have new moment.

Reply:
**Question about the type of startAfter**

> the list should be a [start, end) range.

- number

  Vote for rename it to `startFrom`.

- string

  Vote for keep `startAfter`, due to we only known the `startAfter` key.